app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class CommentsController < ApplicationController
+  def show
+  end
+
+  def new
+    @recording = Recording.find(params[:recording_id])
+    @comment = @recording.comments.new
```

Comment:
Better to do Recording.new in a new - it can unattached in a new - it's just for form_for to infer the url.